### ISSUE TITLE
Change auto accept so groups cannot be laundered

### ISSUE THREAD

**Issue by wingzero54** [CivMC/Civ#146]
Auto accept is a convenient feature meant to make it easy for players to join groups. The problem however is that it also makes it easy to launder groups. If you can find someone with auto accept turned on, you can invite them to your group and set them as primary owner. If they are banned or you otherwise know they won't login, you've now got a group which nobody can discover who it actually belongs to because the owner is a random banned player.

This is obviously unintentional, as the whole point of getting the owner of a group when breaking a snitch, for example, is knowing whose snitch it was. Which is worthless if you've laundered your group and can operate anonymously. 

There are probably a few different ways to go about this. Set auto accept so that it only works if the player is online? Or you could still require them confirming that they were added to a group? On a similar but separate note, an in-game notification that your permission rank has changed on a group would be cool to make it harder to sneak changes in.

**Comment by RedDevel2** [comment]
Would a reasonable enough fix not be to just only have the player join the group once they actually log in?

**Comment by wingzero54** [comment]
Yeah that could work. The main issue is transferring groups to players who haven't logged in, so if you require login to complete auto accept that should do it

**Comment by okx-code** [comment]
Fixed by snitches stating who placed them
